.env

```diff
@@ -0,0 +1,2 @@
+# Only needed on windows: Replace with absolute path to your project
+ABSOLUTE_PATH_TO_INIT=[PATH-TO]oh-my-backend
```

Comment:
Why or rather where is this needed?

Reply:
comment on line 1

Reply:
I read that, but I don't see where this environment variable is being used ...